### PR TITLE
Revise glossary/kops

### PR DESCRIPTION
**Pull request by ydFu**
The reason for the update is as follows:

1. title: 
   - Added full name and same as [kOps repo](https://github.com/kubernetes/kops).
2. full_link:
   - **kubernetes/website** has updated the link address to [`/docs/setup/production-environment/tools/kops/`](https://kubernetes.io/docs/setup/production-environment/tools/kops/)
3. short_description:
   - [The kops official website document](https://kops.sigs.k8s.io/#what-is-kops) has been updated to the description.
4. version description:
   - [The supported version of kops](https://kops.sigs.k8s.io/#what-is-kops) has been updated
5. features:
   - Sync with current kubernetes/website content,The content of the description can refer to [`/docs/setup/production-environment/tools/kops/`](https://kubernetes.io/docs/setup/production-environment/tools/kops/)

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/en/docs/reference/glossary/kops.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
-  * DNS-based cluster identification
-  * Self-healing&#58; everything runs in Auto-Scaling Groups
-  * Limited OS support (Debian preferred, Ubuntu 16.04 supported, early support for CentOS & RHEL)
-  * High availability (HA) support
-  * The ability to directly provision, or to generate Terraform manifests
-
-You can also build your own cluster using {{< glossary_tooltip term_id="kubeadm" >}} as a building block. `kops` builds on the kubeadm work.
+  * Uses DNS to identify clusters
+  * Self-healing: everything runs in Auto-Scaling Groups
+  * Multiple OS support (Debian, Ubuntu 16.04 supported, CentOS & RHEL, Amazon Linux and CoreOS)
```
Comment (chrisnegus) [inline content/en/docs/reference/glossary/kops.md:27]: I'd suggest removing CoreOS. It was end of life in 2020: https://thenewstack.io/say-goodbye-to-coreos/

**Review by chrisnegus — COMMENTED** [review]
Just one comment.

File: content/en/docs/reference/glossary/kops.md (hunk 1 above)
Comment (ydFu) [reply content/en/docs/reference/glossary/kops.md:27]: @chrisnegus 
Thank you for your suggestion, I checked the information and saw other information,as follows:

1. **Add OS support**,see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md):
   - Flatcar: Complete support for Flatcar in [kOps 1.17](https://kops.sigs.k8s.io/releases/1.17-notes/)
     - Support Flatcar: https://github.com/kubernetes/kops/pull/7545
   - Rocky: Add support for Rocky in [kOps 1.23](https://kops.sigs.k8s.io/releases/1.23-notes/)
     - Add support Rocky: https://github.com/kubernetes/kops/pull/13559
2. **Remove OS support**,see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md):
   - CentOS: Remove support for CentOS in [kOps 1.23](https://kops.sigs.k8s.io/releases/1.23-notes/)
     - Remove CentOS: https://github.com/kubernetes/kops/pull/12877
   - CoreOS: Remove support for CoreOS in [kOps 1.19](https://kops.sigs.k8s.io/releases/1.19-notes/)
     - Remove CoreOS: https://github.com/kubernetes/kops/pull/9065
     - [CoreOS EOL](https://thenewstack.io/say-goodbye-to-coreos/)
 3.  All kOps OS support can see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md).

**Comment by ydFu** [comment]
This page is related to `content/en/docs/setup/production-environment/tools/kops.md`, so wait until this PR is merged before continuing

/hold

File: content/en/docs/reference/glossary/kops.md (hunk 1 above)
Comment (ydFu) [reply content/en/docs/reference/glossary/kops.md:27]: I submit a PR(https://github.com/kubernetes/website/pull/36796) to update.

Comment (chrisnegus) [reply content/en/docs/reference/glossary/kops.md:27]: Very nice. Well done.

**Comment by ydFu** [comment]
The features has been updated.(see https://github.com/kubernetes/website/pull/36796).
Please check this update.

/unhold
/cc @chrisnegus @kbhawkey

**Pull request merged**